lib/src/core/config/router/app_router.dart

```diff
@@ -169,5 +169,15 @@ class AppRouter extends $AppRouter {
           page: EditApplicantProfileRoute.page,
           transitionsBuilder: TransitionsBuilders.slideLeft,
         ),
+        CustomRoute(
+            path: '/viewCompanyProfile',
```

Comment:
Can create a new branch for refactor this. Can put path into XPath

Reply:
I will fix it in feature